situp-plugins/elasticsearch/README.md

```diff
@@ -12,6 +12,9 @@ pipeline:
   sink:
     elasticsearch:
       hosts: ["http://localhost:9200"]
+      trace_analytics_raw: true
+      trace_analytics_service_map: false
+      template_file: /your/local/template-file.json
```

Comment:
Maybe a separate PR where we can actually write Readme per usecase. Example,

### Raw trace use case
```
sink:
    elasticsearch:
      hosts: ["http://localhost:9200"]
      trace_analytics_raw: true
      dlq_file: /your/local/dlq-file
      bulk_size: 4
```

### Service-map use case
```
sink:
    elasticsearch:
      hosts: ["http://localhost:9200"]
      trace_analytics_service_map: true
      dlq_file: /your/local/dlq-file
      bulk_size: 4
```
### General use case
```
sink:
    elasticsearch:
      hosts: ["http://localhost:9200"]
      index_alias: "name" # would prefer index instead of index_alias
      index_template: /path/to/index_template.json
      document_id: fieldName # this field will be extracted from the document and used as documentId
      dlq_file: /your/local/dlq-file
      bulk_size: 4
```